compiler/optimizer/LoopReducer.cpp

```diff
@@ -1811,7 +1811,7 @@ TR_Arraytranslate::checkBreak(TR::Block * breakBlock, TR::Node * breakNode, TR::
       //      we also conservatively ensure the range is inside the represented ranges such that we can
       //      add or subtract 1 from the value and still represent it in 2**16 bits for a default value
       //      if we need to generate a table.
-      dumpOptDetails(comp(), "...break tree does not have bconst/cconst/iconst, or not in range - no arraytranslate reduction\n");
+      dumpOptDetails(comp(), "...break tree does not have bconst/iconst, or not in range - no arraytranslate reduction\n");
```

Comment:
shouldn't sconst be in here rather than deleting cconst?

Reply:
The if statement is not checking `cconst` nor `bconst`. Maybe this `dumpOptDetails()` is not up to date. Should I just delete both `cconst` and `bconst`?

Reply:
LoopReducer is not heavily used anymore so I would say if the condition is only checking for iconst just update the details to say as much

Reply:
Fixed.